.github/workflows/backend_build_check.yml

```diff
@@ -0,0 +1,24 @@
+name: build database-service backend
+on: [pull_request]
```

Comment:
Why not also run the tests on `push` events? So you don't need to open a PR but can see the results of the CI

Reply:
because on every push causes a lot of runs. but i can do it on push if u think that's better.

Reply:
I always use push and pull_request, as it will test the code in different contexts, push will test the code as it was pushed, but  pull_request will test the code as it would be after a merge into the target branch. So if there are changes in the target branch which break the new tests, the `push` event test pass but the `pull_request` will fail. And as we have an open source project, we don't have to pay for the many runs (2x4 jobs per pull request).

Reply:
okey.